webapp/channels/src/types/store/plugins.ts

```diff
@@ -187,6 +187,11 @@ export type ProductComponent = {
      */
     mainComponent: React.ComponentType;
 
+    /**
+     * The public component to be displayed when a public route is active.
+     */
+    publicComponent: React.ComponentType | null;
```

Comment:
Curious why the `: React.ComponentType | null` here while other fields are are simply `: React.ComponentType;`.   Not a blocker - just for my own education.